internal/aws/containerinsight/k8sconst.go

```diff
@@ -35,6 +35,11 @@ const (
 	ContainerStatusReason          = "container_status_reason"
 	ContainerLastTerminationReason = "container_last_termination_reason"
 
+	ContainerStatusRunning              = "kube_pod_container_status_running"
```

Comment:
* This file is mostly for k8s consts and labels. Since these are metrics, I would keep these in https://github.com/amazon-contributing/opentelemetry-collector-contrib/blob/main/internal/aws/containerinsight/const.go instead.
* In that other file, the consts would be `StatusRunning`, `StatusTerminated` etc. - the idea being these metrics could be prepended with diff types, container being one of them.
* There was an offline conversation where we had agreed to drop the `kube_pod` from these names to be consistent with all our other metrics - so would be good to update these. 
* This will also fix the issue where the units wont get picked up correctly the way you have it right now (due to the prefix logic) - and customers will also see error logs currently if im not wrong.

Reply:
I left the name as is in the doc currently because I didn't want to really deliberate on that too much while it's up in the air. I can change it. I will also move it over. The top of the other file indicates it's for constants that are both ECS and EKS, which is why I put it in this one instead. Not sure how much we're adhering to that top level comment.

Reply:
On the file, I think what you said makes sense. This file is specific for k8s const and the other one is common for ECS & EKS. These metrics are pretty generic where the status of a container could apply to ECS down the line as well. 
But Im fine leaving them here for now as well.

Reply:
> These metrics are pretty generic where the status of a container could apply to ECS down the line as well.

Yeah, I decided to put it here simply because I was keeping the `kube_pod` prefix since that wouldn't make sense in an ECS context. I will just drop that prefix and move it over. No biggie. I have to do a new rev anyway